src/libraries/Microsoft.PowerFx.Core/Public/Config/SymbolTableOverRecordType.cs

```diff
@@ -40,31 +40,26 @@ internal override bool TryLookup(DName name, out NameLookupInfo nameInfo)
         {
             // Lookup must handle display names
             // This is consistent with RowScope. 
```

Comment:
Do we still need this comment, if we've pushed the handling for display names into TryGetFieldType?

Reply:
There's so much confusion around all this, better to be explicit.